Readme.md

```diff
@@ -41,9 +41,11 @@ import FroalaEditorComponent from 'react-froala-wysiwyg';
 // Import a language file.
 // import 'froala-editor/js/languages/de.js';
 
-// Include font-awesome css if required.
-// import 'font-awesome/css/font-awesome.css';
-// install using "npm install font-awesome --save"
+// Import a third-party plugin.

```

Comment:
@Geet06  You Added this new section can you add how to use these plugins in jsx code?